.github/workflows/continuous_integration.yml

```diff
@@ -11,8 +11,9 @@ jobs:
     name: build
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
Without this, we don't get a chance to see all of the out-of-date package versions all at once.